src/connectors/refinement-list/connectRefinementList.js

```diff
@@ -409,6 +410,42 @@ export default function connectRefinementList(renderFn, unmountFn) {
             .removeDisjunctiveFacet(attributeName);
         }
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`